core/navigation.js

```diff
@@ -20,10 +20,43 @@
 
 goog.provide('Blockly.Navigation');
 
+/**
+ * The current connection the cursor is on.
+ */
 Blockly.Navigation.connection = null;
```

Comment:
Missed last time, but please review which of these should be private and have a trailing _.